cmake/Modules/MacroQbtCommonConfig.cmake

```diff
@@ -95,4 +95,11 @@ macro(qbt_common_config)
         set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
     endif()
 
+    if (CMAKE_GENERATOR MATCHES "Visual Studio")
```

Comment:
Isn't this condition wrong? Shouldn't it be `if (MSVC) ` instead?

Reply:
@glassez You're totally right. Completely missed that.